### ISSUE TITLE
Unable to use, errors in console

### ISSUE THREAD

**Issue by gmyx** [andrewnordlund/backpayCalc#97]
Good day, 

I am trying to use your tool, but it doesn't work. I tried on my phone and work laptop, neither work. I was able to get this error log.

https://andrewnordlund.github.io/backpayCalc/2021/backpayCalc.html?lvl=3&stp=5&strtdt=2016-12-22&enddt=2023-12-28&pdate0=2022-03-14&plvl0=4

backpayCalc.js:1691 Uncaught TypeError: Cannot read properties of undefined (reading 'en')
    at calculate (backpayCalc.js:1691:113)
    at HTMLInputElement.startProcess (backpayCalc.js:512:2)
    at handleHash (backpayCalc.js:316:11)
    at init (backpayCalc.js:174:2)
    at HTMLDocument.getDataFile (backpayCalc.js:2142:3)
calculate @ backpayCalc.js:1691
startProcess @ backpayCalc.js:512
handleHash @ backpayCalc.js:316
init @ backpayCalc.js:174
getDataFile @ backpayCalc.js:2142
backpayCalc.js:1691 Uncaught TypeError: Cannot read properties of undefined (reading 'en')
    at calculate (backpayCalc.js:1691:113)
    at HTMLInputElement.startProcess (backpayCalc.js:512:2)
calculate @ backpayCalc.js:1691
startProcess @ backpayCalc.js:512

**Comment by andrewnordlund** [comment]
So weird!

I'll look into this.

**Comment by andrewnordlund** [comment]
Okay, I found the problem(s):
1. There's a problem with translation when your anniversary date is on Dec. 22.  Ooopsies.
2. If you update the date to another date and re-run, it's not properly resetting an associative array to an initial state, so the "null"s that show up in the Results table persist.

I've made a few other changes which require some translation.  As soon as I fix these issues and get the translations done, I'll post the new versions.
